packages/core/src/styles/tokens/tokens.stories.ts

```diff
@@ -4,162 +4,1018 @@
   return html`
-    <div cds-layout="vertical gap:xl">
-      ${getTokensByCategory('global').map(
-        token => html`<p cds-text="subsection">${token.cssProp}: ${getTokenPxValueTemplate(token)}</p>`
-      )}
+    <style>
+      .cds-object-app-demo {
+        background: var(--cds-alias-object-app-background);
+      }
+
+      .cds-object-container-demo {
+        background: var(--cds-alias-object-container-background);
+        border: var(--cds-alias-object-border-width-100) solid var(--cds-alias-object-container-border-color);
+      }
+
+      .cds-object-overlay-demo {
+        position: relative;
+      }
+
+      .cds-object-overlay {
+        background: var(--cds-alias-object-overlay-backdrop-background);
+        opacity: var(--cds-alias-object-opacity-300);
+        position: absolute;
+        left: 0;
+        top: 0;
+        bottom: 0;
+        right: 0;
+      }
+
+      .cds-object-overlay-content {
+        position: relative;
+        background: var(--cds-alias-object-overlay-background);
+      }
+    </style>
+
+    <div class="cds-object-app-demo" cds-layout="vertical gap:lg align:stretch">
+      <h3 cds-text="section">App</h3>
+      <p cds-text="body">--cds-alias-object-app-background</p>
+      <div class="cds-object-container-demo" cds-layout="vertical gap:lg p:lg">
+        <h3 cds-text="section">Container</h3>
+        <p cds-text="body">--cds-alias-object-container-background</p>
+        <p cds-text="body">--cds-alias-object-container-border-color</p>
+      </div>
+      <div cds-layout="p:lg" class="cds-object-overlay-demo">
+        <div class="cds-object-overlay"></div>
+        <div cds-layout="vertical gap:lg p:lg" class="cds-object-overlay-content">
+          <h3 cds-text="section">Overlay</h3>
+          <p cds-text="body">--cds-alias-object-overlay-background</p>
+          <p cds-text="body">--cds-alias-object-overlay-backdrop-background</p>
+          <p cds-text="body">--cds-alias-object-opacity-300</p>
+        </div>
+      </div>
+    </div>
+  `;
+};
+
+export const objectDemo = () => {
+  return html`
+    <style>
+      .cds-card-mock {
+        background: var(--cds-alias-object-container-background);
+        border: var(--cds-alias-object-border-width-100) solid var(--cds-alias-object-container-border-color);
+        border-radius: var(--cds-alias-object-border-radius-100);
+        box-shadow: var(--cds-alias-object-shadow-100);
+        padding: var(--cds-global-space-8);
+        color: var(--cds-global-typography-color-500);
+        max-width: 350px;
+      }
+    </style>
+    <div class="cds-card-mock" cds-layout="vertical gap:lg">
+      <h3 cds-text="section">Mock Card Component</h3>
+      <p cds-text="body">This is a mock card component made with Object tokens.</p>
+      <cds-button action="outline" cds-layout="align:right">Save</cds-button>
     </div>
   `;
 };
 
-export const space = () => {
+export const interactionMenuDemo = () => {
```

Comment:
These demos are for the interaction token docs and would slowly be replaced by the real component. This just demonstrates the tokens work for our use cases consistently across different components.